datapackage_pipelines_measure/processors/add_timestamp.py

```diff
@@ -0,0 +1,26 @@
+import datetime
+
+from datapackage_pipelines.wrapper import process
+
+from datapackage_pipelines_measure.config import settings
+
+import logging
+log = logging.getLogger(__name__)
+
+
+def modify_datapackage(datapackage, parameters, stats):
+    datapackage['resources'][0]['schema']['fields'].append({
+      'name': 'timestamp',
+      'type': 'datetime',
+      'format': '{}'.format(settings.TIMESTAMP_DEFAULT_FORMAT)
+    })
+    return datapackage
+
+
+def process_row(row, *args):
+    now = datetime.datetime.now().strftime(settings.TIMESTAMP_DEFAULT_FORMAT)
```

Comment:
Do we want all timestamps in the same processing batch to be slightly different?
I think it's better to calculate `now` one and use the same value in `process_row`.